main.go

```diff
@@ -504,10 +504,13 @@ func main() {
 					return fmt.Errorf("admin user creation failed: %w", err)
 				}
 
+				token := "EST" + uuid.New().String() + "ARY"
 				authToken := &util.AuthToken{
-					Token:  "EST" + uuid.New().String() + "ARY",
-					User:   newUser.ID,
-					Expiry: time.Now().Add(time.Hour * 24 * 365),
+					Token:     token,
+					TokenHash: util.GetTokenHash(token),
+					Label:     "admin",
+					User:      newUser.ID,
+					Expiry:    time.Now().Add(time.Hour * 24 * 365),
```

Comment:
Let’s put the expiry value on the constants.go and use it across the estuary code base.